File: Back-end/src/main/java/com/example/demo/entity/ClientExperienc.java
package com.example.demo.entity;

import javax.persistence.Basic;
import javax.persistence.Entity;
import javax.persistence.FetchType;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.Lob;
import javax.persistence.OneToOne;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class ClientExperienc {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;
	private Long beforekilo;
	private Long afterkilo;
	private String name;
	private Long week;
	private Long fatbefor;
	private Long fatafter;
	private String comextern;
	@Lob
	@Basic(fetch = FetchType.LAZY)
	private byte[] image; 
	@Lob
	@Basic(fetch = FetchType.LAZY)
	private byte[] image2;
	@OneToOne
	@JoinColumn(name = "userRoom")
	private Room userRoom;
}

File: src/app/ClientExperienc.ts
export class ClientExperienc {
    id: number;
    image:String;
    image2:String;
    beforekilo:number;
    afterkilo:number;
    name:String;
    week:number;
    fatbefor:number;
    fatafter:number;
    comextern:String;
        constructor(data: any = {}) {
        this.id= data.id|| 0;
        this.image = data.image;
        this.image2 = data.image2;
        this.beforekilo = data.beforekilo || 0;
        this.afterkilo = data.afterkilo || 0;
        this.name = data.name || "";
        this.week = data.week || 0;
        this.fatbefor = data.fatbefor || 0;
        this.fatafter = data.fatafter || 0;
        this.comextern = data.comextern || "";
}
}